src/main.c

```diff
@@ -2174,6 +2180,49 @@ void on_middle_click() {
     }
 }
 
+int isDefinedCraftKey(char* str, int start, int end)
+{
+    char str2[end - start];
+    for (int i = 0; i < (end - start); i++)
+    {
+        str2[i] = str[start + i];
+    }
+    int tru = 1;
+    if (str2[0] != 'C')
```

Comment:
Instead of using multiple if statements, you can you single if statement  like if(str2[0] != 'C' && str2[1] != 'R' && str2[2] != 'A' && str2[3] != 'F' && str2[4] != 'T')